### DISCUSSION TITLE
Can I make it so DistroBox won't create a desktop entry for a container if it already exists?

### DISCUSSION THREAD

**Discussion by poperigby** [89luca89/distrobox#1112] [Q&A] [answered]
I have a custom desktop entry for my container, and it gets overwritten every time I recreate it.

**Comment by 89luca89** [comment, marked as answer]
use the `--no-entry` flags, see the usage of `distrobox create`

**Reply by poperigby** [reply]
Can I use that with `distrobox assemble`? It doesn't seem to work when I added it as `no_entry=true`.

**Reply by 89luca89** [reply]
you need to set `entry=false`

**Reply by poperigby** [reply]
Perfect, thanks :slightly_smiling_face:!

**Reply by poperigby** [reply]
@89luca89 Setting entry to false deletes my desktop entry when I run `distrobox assemble` with this configuration file:

```ini
[toolbox]
image=ghcr.io/poperigby/toolbox:latest
pull=true
replace=true
entry=false
```

How can I get around this?
